control-plane/tests/test_cli_inspection.py

```
@@ -40,6 +40,36 @@ def _load_wazuh_fixture(name: str) -> dict[str, object]:
+        store: object | None = None,
+        host: str | None = None,
+        port: int | None = None,
+    ) -> tuple[object, AegisOpsControlPlaneService, CaseRecord, str, datetime]:
+        if store is None:
+            store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host="127.0.0.1" if host is None else host,
+                port=0 if port is None else port,
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)